packages/postgres-database/tests/test_services_consume_filetypes.py

```diff
@@ -164,7 +164,7 @@ async def test_contraints(conn):
             ]
         )
         .where(services_consume_filetypes.c.filetype == "DCM")
-        .as_scalar()
+        .scalar_subquery()
```

Comment:
Is this also a change in the api? Can you provide in the PR with a good reference link that stresses out the changes?

Reply:
yes deprecation warning